ui/next/app/redux/uiData.ts

```diff
@@ -153,16 +153,22 @@ export function loadUIData(...keys: string[]) {
     dispatch(fetchUIData());
 
     return getUIData({ keys }).then((response) => {
-      response.getKeyValues().forEach((val, key) => {
+      let keyValues = response.getKeyValues();
+
+      _.each(keys, (key) => {
+        let decoded: any = null;
```

Comment:
If you want to have an explicit `setUIDataKey(key, undefined)` to make the not-exists case more explicit, then you don't need to define `decoded` anymore.  Example:

`` typescript
let bb = // get key value if it exists
if (bb) {
  let str = bb.readString(bb.limit);
  if (str) {
    dispatch(setUIDataKey(key, JSON.parse(str));
    return;
  }
  dispatch(setUIDataKey(key, undefined));
}